servers/v3-proxy-api/src/test/fixtures/fetch.ts

```diff
@@ -14,8 +14,8 @@ export const expectedFetch: FetchResponse = {
   },
   cachetype: 'db',
   list: {
-    '11231399273': {
-      item_id: '11231399273',
+    '2111834840': {
```

Comment:
just changed the item_id of one of the fixtures (and left the resolved_id alone)